### PR TITLE
Fixed some remarks

### PR DESCRIPTION
**Pull request by vrezhar**
Email still won't work

### CODE REVIEW

File: grails-app/controllers/com/secured/auth/UserController.groovy [hunk 1]
```diff
@@ -36,37 +37,45 @@ class UserController  {
...
+            redirect(action:"register")
+            return
         }
-        flash.model = cmd
-        redirect(action:"register")
-
+        if(!userValidator.isPasswordValid(cmd))
+        {
+            flash.model = cmd
+            redirect(action:"register")
```
Comment (edasatryan) [inline grails-app/controllers/com/secured/auth/UserController.groovy:54]: do not send model by flash 
redirect(action:"register", model :[cmd: cmd])

Comment (edasatryan) [reply grails-app/controllers/com/secured/auth/UserController.groovy:54]: The same for the others

File: grails-app/domain/com/secured/auth/commands/UserCommand.groovy [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.secured.auth.commands
```
Comment (edasatryan) [inline grails-app/domain/com/secured/auth/commands/UserCommand.groovy:1]: Please move command objects to package
/src/main/groovy/com/secured/user 
and implemnt from Validateable to have validatable command 
Like:  class XXXCO implements Validateable {

File: grails-app/controllers/com/secured/auth/UserController.groovy (hunk 1 above)
Comment (vrezhar) [reply grails-app/controllers/com/secured/auth/UserController.groovy:54]: redirect methodin map poxanceluc inqy mapum model parametri chi spasum ughaki, dra hamar tenc ban chem arel, lavaguyn depqum karam senc grem redirect(action: "register", params: params) u register actionum grem UserCommand cmd = UserCommand.create(params) kam cmd-n tam vorpes parameter

**Pull request closed without merging**